Assets/HoloToolkit/UX/Scripts/Lines/LineParticles.cs

```diff
@@ -0,0 +1,135 @@
+﻿//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+//
+using UnityEngine;
+
+namespace MRTK.UX
+{
+    public class LineParticles : LineRendererBase
+    {
+        const int GlobalMaxParticles = 2048;
+        const float GlobalParticleStartLifetime = 0.5f;
```

Comment:
private or public accessors?

Reply:
I don't think they're needed since they're const & can't be modified.

Reply:
I think it's always good to know, that way future devs know what scope they're in.

Reply:
I'm a little confused because const values can't use get / set accessors. Are you thinking we should change these to public properties that return constant values?

Reply:
It's just helpful to know if this information should have the _ability_ to be accessed outside the scope of just this class.  Idk it's really not a big deal here. Just my 2c

Reply:
No worries, I just use const all the time so I wanted to get on the same page. If you post an 'ideal scenario' I'll modify the class.